examples/fungible_token/src/lib.rs

```diff
@@ -0,0 +1,309 @@
+#![no_std]
+#![feature(const_btree_new)]
+
+use codec::{Decode, Encode};
+use gstd::{debug, exec, ext, msg, prelude::*, ProgramId};
+use primitive_types::H256;
+use scale_info::TypeInfo;
+
+const GAS_RESERVE: u64 = 50_000_000;
+
+#[derive(Debug)]
+struct FungibleToken {
+    name: String,
+    symbol: String,
+    total_supply: u128,
+    balances: BTreeMap<ProgramId, u128>,
+    allownces: BTreeMap<ProgramId, BTreeMap<ProgramId, u128>>,
+}
+
+static mut FUNGIBLE_TOKEN: FungibleToken = FungibleToken {
+    name: String::new(),
+    symbol: String::new(),
+    total_supply: 0u128,
+    balances: BTreeMap::new(),
+    allownces: BTreeMap::new(),
+};
+
+impl FungibleToken {
+    fn set_name(&mut self, name: String) {
+        self.name = name;
+    }
+    fn name(&self) -> String {
+        self.name.clone()
+    }
+    fn set_symbol(&mut self, symbol: String) {
+        self.symbol = symbol;
+    }
+    fn symbol(&self) -> String {
+        self.symbol.clone()
+    }
+    #[allow(dead_code)]
+    fn total_supply(&self) -> u128 {
+        self.total_supply
+    }
+    #[allow(dead_code)]
+    fn decimals(&self) -> u8 {
+        18
+    }
+    // TODO: Use checked math and return error if Overflow.
+    fn increase_total_supply(&mut self, amount: u128) {
+        self.total_supply += amount;
+    }
+    // TODO: Use checked math and return error if Underflow.
+    fn decrease_total_supply(&mut self, amount: u128) {
+        self.total_supply -= amount;
+    }
+    fn set_balance(&mut self, account: &ProgramId, amount: u128) {
+        self.balances.insert(*account, amount);
+    }
+    fn get_balance(&self, account: &ProgramId) -> u128 {
+        *self.balances.get(account).unwrap_or(&0u128)
```

Comment:
The suffix is not needed.
```suggestion
        *self.balances.get(account).unwrap_or(&0)
```